webroot/js/cake-core.js

```diff
@@ -0,0 +1,101 @@
+var postLinkFunction = function () {
```

Comment:
I this JS should NOT be in the core and instead there should be a basic XHR call which loads a delete view (GET) which has a confirm button). If JS is disabled the whole framework will work unobtrusively and the only JS required is a general js handler that loads get requests as XHR HTTP requests instead of regular HTTP requests (a very few lines that are useful otherwise for RAD as well).